spec/serializers/firm_serializer_spec.rb

```diff
@@ -40,6 +40,14 @@
       expect(subject[:investment_sizes]).to eql(firm.investment_size_ids)
     end
 
+    it 'exposes `advises_on_investments`' do
+      expect(subject[:advises_on_investments]).to be
+    end
+
+    it 'exposes `investment_transfers`' do
+      expect(subject[:investment_transfers]).to be
+    end
+
```

Comment:
Unless you're talking about both cases, I would be more likely to agree about the first case. But even still, we'd just be checking that rails works correctly, IMO.